apps/trading/e2e/tests/wallet/test_wallet.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import re
+import json
+from playwright.sync_api import Page, expect, Route
+from vega_sim.service import VegaService
+from conftest import init_vega
+from fixtures.market import setup_continuous_market
+
+order_size = "order-size"
+order_price = "order-price"
+place_order = "place-order"
+order_side_sell = "order-side-SIDE_SELL"
+market_order = "order-type-Market"
+tif = "order-tif"
+expire = "expire"
+
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    return setup_continuous_market(vega)
+
+def handle_route_connection_lost(route: Route, request):
```

Comment:
So if my assumption is correct, here we are just handling incoming API requests. I would personally put these into a specific handler file and would not bundle it in with the tests.

Reply:
I'm not sure im sold on pulling into a separate file, as currently these tests are the only place we do anything like this, and not sure we gain a lot from pulling into a separate file vs keeping here?

Reply:
Yeah cool. I normally do that so it is more visibly reusable if it is _likely_ to be useful to someone else or reused. However if those things aren't true then no issue with it being left